### DISCUSSION TITLE
No longer seem to be getting motion/ding alerts

### DISCUSSION THREAD

**Discussion by thefunkygibbon** [tsightler/ring-mqtt#896] [Support Q&A]
I've been using it successfully for about a year and this last week or so has been super sketchy in terms of if i get alerted that someone is at the door or motion recording or anything of the link. 

I've checked the entity within Home assistant and it shows that I havent had any motion or dings for days (the default reboot time/date).
Testing moving around outside and it doesnt pick anything up. 

Weirdly it must be working with MQTT correctly because I can happily click on the take snapshot button and it updates the snapshot, which , as well as the stream, I can view with no issues.

Any thoughts?
Obviously things have been rebooted to no avail.

Oh and the Ring HA plugin which I still have setup shows the motion detected and dings donged correctly.

**Comment by thefunkygibbon** [comment]
just tested another mqtt based device and that is working fine. 

whilst keeping monitoring the mqtt queues I'm not seeing anything when i ring the bell or move in front of it.  but if i press the take snapshot button i see the message in mqtt about it.

**Comment by thefunkygibbon** [comment]
just looking through the FAQ stuff for this specific issue and it says to log into the ring-mqtt web UI ... i didnt know there was one and I Cannot see any documentation as to how to do so

**Reply by tsightler** [reply]
The TL;DR version is, stop ring-mqtt, remove ring-mqtt accounts from Ring Control Center, re-auth ring-mqtt as you did during initial installation (process varies based on install method, Web UI is primarily for addon users and button to launch it is in the HA UI).

**Reply by thefunkygibbon** [reply]
> The TL;DR version is, stop ring-mqtt, remove ring-mqtt accounts from Ring Control Center, re-auth ring-mqtt as you did during initial installation (process varies based on install method, Web UI is primarily for addon users and button to launch it is in the HA UI).

got ya, thanks for replying.   
fwiw I did reconnect it again last night, but I couldn't seem to remove the accounts from the control center (although I just tested it again and i can now)   and it didnt make any difference.    i dont have anything listed saying ring-mqtt or anything similar,  its all "Python  android" and a couple of "device name not found" which are all dated long before i started using this system. 
I'll go through and disconnect everything i guess.   
but since I've got ring normal integration , i've just swapped the automations i have to use those motion sensor entities instead, and tbf everything seems to be working as expected.

**Reply by tsightler** [reply]
Python is probably the native HA integration so you don't want to remove those if you are using it as well.  It's only the ones that say ring-mqtt, and perhaps blank ones, that have to be removed.

However, it's a good idea to remove any unused devices from this screen because having lots of phantom/unused devices is one of the most common causes of notification issues.

**Reply by FermionAlpha** [reply]
> The TL;DR version is, stop ring-mqtt, remove ring-mqtt accounts from Ring Control Center, re-auth ring-mqtt as you did during initial installation (process varies based on install method, Web UI is primarily for addon users and button to launch it is in the HA UI).

I was having motion events not arrive or at best arrive after several minutes. These steps fixed the problem for me. Thank you!

For some reason I had dozens of authorized devices in the Ring Control Center.

**Comment by jasonhnz** [comment]
I had this issue as well. I logged into control centre ad only one ring-mqtt account logged in so removed that and some others. 
After a restart, I now have motion events again but even hough there have been no changes to my cameras, all smart alerts no longer exist. There are no options for person detection and all the sensors have been removed. Has something happened on ring's side ? Is there something else I could try to try and get smart events/sensors back?

**Reply by tsightler** [reply]
I don't really know what this means.  Smart alerts is a feature in the Ring app, available only with a Ring subscription.  Whether it exists or not has nothing to do with ring-mqtt, nor do I have any idea what "all the sensors have been removed" means.

**Comment by Kesslermeinname** [comment]


**Reply by tsightler** [reply]
> Is there any way to fix this issue for good?

I find this a funny question since we don't really know what the problem is.  It's like asking "my car won't start, any way to fix the issue for good?"  The only way to know the answer is to determine the source of the problem.

You state that you had to "delete the MQTT client", but do you know for sure that simply restarting ring-mqtt addon doesn't solve the problem?  That would tell us a good bit about the issue. 

Basically, there is no way to magically fix such issues, somebody has to put in the work/time to track down the actual cause and, since I'm unable to reproduce the problem with my setup no matter how hard I have tried (my push notifications haven't failed in months), somebody who has the problem has to commit their time.  I'm willing to work with someone on it, but 99% of people just post here about some random issue and never respond again when they realize they might need to commit some significant amount of time to resolve it.

My gut is that it's some issue with push-receiver, which makes this even more complicated since that is an upstream dependency.  However, I really don't know why, for example, thousands of people seem to have almost no problem, while a small percentage of people have regular issues.  It could be firewalls that timeout the connection, could be some message format causing push-receiver to crash, could be token expiration not being handled properly, who knows, but it requires **_deep_** investigation to have any chance to resolve.

**Reply by jeffgraham-hub** [reply]
@tsightler
 I don't mean to clutter this thread and if you prefer I can create a new one. I have had an issue several times recently where I have lost motion notification for cameras only. Up to now I havent really noted the frequency. I am willing to try and assist you in identifying the cause. For me a restart of the addon does fix the problem. Is there anything you need soecifically or just start capturing logs until it happens again?

**Reply by tsightler** [reply]
@jeffgraham-hub Thanks for the offer, the biggest issue is, well, I don't know what we need to do because it's not even slightly clear where the problem is.  However, the fact that restarting the addon does fix it at least provides some clue that push-receiver the culprit.

The problem as I see it is that push-receiver is not very well instrumented.  I know there was one crash that was fixed in the latest version, I hope we'll push a new version of ring-client-api and then ring-mqtt, that includes this sometime over the weekend.  But, assuming it happens again, we'd need logs, but also, ideally, a login to the container to check the established network connections and, in a perfect world, a network trace while the problem is happening.

If you can do all of this great, if not, I'll probably need to build custom version of ring-client-api and push-receiver that instrument some of this so that we have any chance to figure out what is happening.  If you are willing to commit some time over the next months, I'd say we should open a new issue and work on it long term.

**Comment by madtraxx18** [comment]
@tsightler 

I’m experiencing the same issue with my doorbell 3 pro which just doesn’t seem to update the motion sensor.

steps I’ve taken so far;

rebooted the plugin 
Removed all partnerships in the ring backend
Reauthenticated 

still the same issue, I can jump up and down in front of the camera but it never detects motion in mqtt.

I’m willing to help you debug this, except the current HA supervisor update has made logging for all addons unavailable in the GUI.

One thing I would like to point out is that the WiFi connection of my doorbell is really crappy (around -80 rssi)

All other ring cams (indoor non battery ones) are working fine with motion detection.

As I’ve read in another topic it might be related to a loss of connectivity to the device.

**Reply by tsightler** [reply]
If it's just for one camera/doorbell that is a really tough one as, from the perspective of ring-mqtt, all notifications are delivered via a single connection to Firebase messaging servers, there's nothing unique per-device.  Do you get notifications in the Ring app?  Do you use any features like Smart Alerts, and are motion notifications actually turned on in the app (can you try toggling them off/on)?  What about ding notifications, do they work for this camera if someone presses the bell?

**Reply by tsightler** [reply]
Also, you'll want to check if motion detection is enabled in the configuration options of the camera in HA.  I've been a bit surprised to find that several people didn't realize that turning off "Motion Detection" actually does what it says and turns of motion detection!  :)

**Comment by madtraxx18** [comment]
![IMG_0004](https://github.com/user-attachments/assets/b563c0a6-6f4d-4fb3-bc5d-f04775baa8fe)

notifications are disabled, smart notification are enabled and only allowed for packages within a specific zone. 

If I enable notifications I do receive notifications in the app.

motion detection is enabled both in the app and HA

**Reply by madtraxx18** [reply]
Edit: ding sensor does update when pressed

**Reply by tsightler** [reply]
> smart notification are enabled and only allowed for packages within a specific zone.

Can you clarify what you mean when you say "only allowed for packages"?  Does this mean you've used Smart Alerts to disable notifications for other events?  Smart Alerts are set for the device, so if you disable notifications here, then ring-mqtt won't receive event notifications either.

**Comment by madtraxx18** [comment]
Issue clarified, enabled notifications in smart events, disabled notifications in the main setting. 

Now the sensor is updating upon movement (this setting was already in place for the other cameras)

so the motion sensor doesn’t update when there are no notifications enabled in smart notifications despite that the device still registers these events  on the timeline of the recording.

so camera wise they should be enabled, app wise they are disabled (at least that’s how I’ve now configured my setup) this prevents the notifications from popping up on my phone as a push notification but it still allows the plugin to receive the update of movement.

**Comment by Kuhli79** [comment]
Hi everyone,

I'm facing exactly the same issue described here, but in my case, it affects **all** my cameras and doorbells.

I've already performed multiple troubleshooting steps, including:

* Restarting both Ring2MQTT and Home Assistant multiple times.
* Completely removing Ring2MQTT (full uninstall) and then reinstalling and re-authenticating it from scratch.
* Removing and re-adding the integration completely, including clearing out all authorized devices in Ring Control Center.
* Even fully removing the official Home Assistant Ring integration to avoid possible conflicts.

Unfortunately, none of these steps made any difference. No motion or ding events are showing up at all. Occasionally, I see log entries like "new event 1 motion stream detected, URL updated," but despite this, the actual MQTT `binary_sensor` for motion has **never** changed state since initial setup. It stays permanently "off".

If anyone has further suggestions or needs specific logs to help narrow this down, please let me know. I'm happy to help troubleshoot further.

Thanks a lot!

**Comment by artistan82** [comment]
Hi everyone,

Encountering similar issue with @Kuhli79.
I have two installations of ring-mqtt on two different location.
First installation works with no issue. The new second installation (this uses a different ring account) is not receiving any motion/ding alert.

Ran through all the mention troubleshooting steps, including:

- Removing all ring-mqtt and unamed devices from authorized list.
- Reauthenticating ring-mqtt.
- Restarting Ring2MQTT and Home Assistant multiple times.

I do see log entries like "new event 1 motion stream detected, URL updated" just like @Kuhli79 but the binary sensors for motion always stays off.

The first installation is HA, Mosquitto MQTT and Ring-MQTT on a server with individual docker.
The second installation is on a Raspberry Pi with the HASS OS, MQTT and Ring-MQTT as addon.

I've tried the second account on the first installation and encountering the same issue on the first installation.

**Reply by tsightler** [reply]
> I've tried the second account on the first installation and encountering the same issue on the first installation.

So what does this tell you?  It's an account issue, not an issue with ring-mqtt.  Almost certainly some device/account setting is not allowing motion detection.  Are you sure you don't have smart alerts or some other feature that will suppress alerts?  Sometimes just toggling motion warnings off for 30 seconds and back on will help.

**Reply by artistan82** [reply]
The problem is I’ve tried all that steps already.
I’m kind of at a lost since nothing obvious is showing up in the logs.

The only thing I feel that might be relevant to the issue I see is this two entry after re-authenticating ring-mqtt.
![IMG_4141](https://github.com/user-attachments/assets/828d1bd2-823d-4bc4-9d3b-4bd6092e8122)

**Reply by tsightler** [reply]
So you explicitly tried toggling "Ring Alerts" and "Motion Alerts" off on this account in the Ring app, waiting at least 30 seconds, and the toggling them back on?

Also, you have explicitly followed the steps, i.e. stop ring-mqtt completely, remove all authorized devices from Ring Control Center, restart ring-mqtt, re-authenticate.

Overall, the above messages indicate that the account in question is unable to subscribe to events, it's the equivalent of the Ring Alerts/Motion Alerts setting being disabled in the app.

The next suggestion would be to try a different account.

**Comment by dacootmeister** [comment]
Got sort of the same issue as @artistan82 and @Kuhli79 . 
Both My Chime Pro v2 dont't work anymore when the Battery Doorbell Plus is pressed. I dont use the motion alerting. 
When I disable the Ring2MQTT plugin the chimes respond as normal.

For me, and the family, this is clearly a bug although I cant find anything out of the ordinary in the debug log.  See attachhed
[03cabcc9_ring_mqtt_2025-07-04T12-51-59.572Z.log](https://github.com/user-attachments/files/21058480/03cabcc9_ring_mqtt_2025-07-04T12-51-59.572Z.log)

**Reply by tsightler** [reply]
> I don't use the motion alerting.

You will never get motions in ring-mqtt if you don't use motion alerting so I have no idea what this means.  I have no idea how ring-mqtt could possibly impact chime settings, it makes no changes to settings at all by default.

**Reply by dacootmeister** [reply]
I dont need/use motion alerting. Fact is that along the last couple of months *something* has changed (not in my setup) in code, or Ring API which caused the bell when activated to not be able to connect with the chimes.
For me this was a hell of a search and replaced my bell twice because of it :-(
So hopefully someone can help with debugging and until then, I will no longer use this integration, which is a pity as it surely provided great features.

Thanks @tsightler for notifying that the native integration now also added live streams. I use this feature in my automations.

**Reply by tsightler** [reply]
Do you mean your physical/electronic doorbell doesn't chime?  I don't think there's any way ring-mqtt itself could cause this, it's just a client like logging into the phone app.  However, you mention automations  with live streams, which makes me wonder if you are using some automation to record the stream when a ding is detected, or perhaps with some other trigger?  If so, then I would suspect this is the reason the chime doesn't work.

**Reply by dacootmeister** [reply]
Electronic doorbell = my Chime Pro's 2n generation. Anyway, it is related as the doorbell vs chime doesnt respond properly when the client/plugin is enabled. 
It works flawless when the plugin is diabled. So I nw reverted to the native integration which does exactly what I need, and that is a camera view

**Comment by thefunkygibbon** [comment]
i gave up and am having to just use the actual ring integration for motion alerts and just use this addon specifically for pulling the video streams.

**Reply by tsightler** [reply]
I thought they added live streams to the native integration some time ago.  This is why I've been considering pulling the camera support completely anyway, because it mostly just duplicates the functions of the native Ring integration.

The concern I have is that, as far as I can tell, the implementation is just basically copied from the work we've done.  I have no idea why motion detection would work in the native integration and not here since it's literally just a port of the code from ring-client-api.

That being said, it still works for 99.9% of users, and I have no way to troubleshoot it for those where it doesn't work so no much to do until somebody that has some development knowledge has the problem and is willing to dig in.

**Reply by thefunkygibbon** [reply]
news to me!  i've just checked on my Ring doorbell entities and neither allow me to connect to live streams.  Maybe it requires a subscription or something? I no longer subscribe as a) it was getting more and more expensive each time it renewed and b) all my usecases were addressed with using this addon.   
FWIW when i try to access the "live view" within the doorbell object i just get "Failed to connect WebRTC stream: Answer tried to set recv when offer did not set send"

**Reply by omian** [reply]
@thefunkygibbon based on the native integration documentation, access to last recording requires a subscription but live view should be available for all: https://www.home-assistant.io/integrations/ring/#camera
I have both entities but I have a subscription. If you don't have access to live view might be worth providing feedback on the documentation.

**Comment by tlanglois1111** [comment]
I would add to this thread that you should verify that your scheduling isn't turning off motion alerts.  You can still have motion detection turned on but have alerts turned off by a schedule, which means no motion detection in HA.  I assumed at first that simply leaving detection on would provide those events to HA, but the actual alerts have to be on as well.

**Comment by babbiesbutt** [comment]
Hey there Ring-MQTT GitHub community,
Meee-too. I was able to successfully setup my HA (on Synology 920+ NAS) with Ring, MQTT add-on & Ring-MQTT addon. Then after everything seemed OK, I added Go2RTC for external RSTP streams. I also added MWTT Explorer along the way, and very recently Stopped that (not uninstalled). Though somewhere along the way all my Motion Detections & Ding (Ring button-push) vanished. I've tried all of the above (removed credentials on Ring online, Stop, Started all add-ons, reauthorized Ring & Ring-MQTT and rebooted HA multiple times just to be sure I was not nuts.
Yes, I even turned-off Smart motion detection in my Ring app. I also tried Turning-off the Motion detection in my Ring app for 30+ sec, then re-enabled.....NO SUCCESS - still no Motion Detection or Ding.
Bottomline - unless I "Stop" the Ring-MQTT, my Ring Motion detection & Ding (button-push) are not available on-going daily.

Any further thoughts on this issue?
I'm mucho-hours into this and cannot seem to find a reasonable solution.
HELP !

**Reply by babbiesbutt** [reply]
oops - MQTT (not MWTT)

**Comment by kmanan** [comment]
Me too. I have a doorbell and some cameras and none of them register any motion. It's just stuck in "clear"
